code/modules/reagents/newchem/patch.dm

```diff
@@ -0,0 +1,18 @@
+/obj/item/weapon/reagent_containers/pill/patch
+	name = "chemical patch"
+	desc = "A chemical patch for touch based applications."
+	icon = 'icons/obj/chemical.dmi'
+	icon_state = "bandaid"
+	item_state = "bandaid"
+	possible_transfer_amounts = null
+	volume = 50
+	apply_type = TOUCH
+	apply_method = "apply"
+	apply_method_plural = "applies"
```

Comment:
If anything, this should inherit brute packs, because this is our version of pre-loaded patches with medicine

Inheriting pill is extremely dumb and will cause problems